spec/core/ics-002-client-semantics/README.md

```diff
@@ -337,6 +337,25 @@ Once misbehaviour is detected, clients SHOULD be frozen so that no future update
 A permissioned entity such as a chain governance system or trusted multi-signature MAY be allowed
 to intervene to unfreeze a frozen client & provide a new correct ClientMessage which updates the client to a valid state.
 
+#### Retrieve client status 
+
+Status is an opaque function defined by a client type to retrieve the current clientState. Status can be either `Active`, `Expired`, `Unknown` or `Frozen`.
```

Comment:
similarly here, If `Unknown` status is not needed in the wasm-client we can remove it from client-semantics too.